.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+        ["babel-preset-moxy/lib", {
+            "lodash": false }
+        ]]
+}
```

Comment:
Since this is a CLI tool, I think it's debatable if we need to transpile it or not. What do you think? If we opt to not transpile it, then we should use the source code in a `lib/` folder instead of `src/`.

Paging @satazor.

Reply:
It's not being transpiled by babel. It's just using the import/export functionality due to the `esm` package.

Relevant info tho:
https://gist.github.com/tracker1/59f2c13044315f88bee9